pkg/apis/kops/topology.go

```diff
@@ -49,3 +52,11 @@ const (
 	DNSTypePublic  DNSType = "Public"
 	DNSTypePrivate DNSType = "Private"
 )
+
+type RouteSpec struct {
+	// The IPv4 CIDR block used for the destination match.
+	CIDR *string `json:"cidr,omitempty"`
+
+	InternetGateway *string `json:"internetGateway,omitempty"`
+	NatGateway      *string `json:"natGateway,omitempty"`
```

Comment:
You might be able to get away with "Destination *string `json:"destination,omitempty"` because then it's a little more cloud-agnostic.  On the other hand, what you have is clearer.

Reply:
In addition to internet gateway and nat gateway, this should also support peering connections.